### PR TITLE
Bug 1121343 - [FTE] Notification indicator shows during FTE, but un-acti...

### PR DESCRIPTION
**Pull request by mancas**
...onable

### CODE REVIEW

**Comment by try-server-hook** [comment]
[mancas Manuel Casas (mancas)](https://github.com/mancas) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c8e25461e87a)

File: apps/system/js/notifications.js [hunk 1]
```diff
@@ -641,6 +642,11 @@ var NotificationScreen = {
 
   updateNotificationIndicator: function ns_updateNotificationIndicator() {
     if (this.unreadNotifications.length) {
+      // If the ftu is running we should not show the ambient indicator
+      // because the statusbar is not accessible
+      if (FtuLauncher.isFtuRunning()) {
```
Comment (etiennesegonzac) [inline apps/system/js/notifications.js]: you should be able to use `Service.query('isFtuRunning')` here (more future-proof :))

**Comment by try-server-hook** [comment]
[mancas Manuel Casas (mancas)](https://github.com/mancas) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f0fd9ec89e6a)

**Comment by try-server-hook** [comment]
[mancas Manuel Casas (mancas)](https://github.com/mancas) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f4e477981963)

File: apps/system/test/unit/notifications_test.js [hunk 2]
```diff
@@ -268,6 +269,22 @@ suite('system/NotificationScreen >', function() {
       UtilityTray.shown = false;
     });
 
+    test('should not show ambient indicator if FTU is running', function() {
+      this.sinon.stub(Service, 'query', function(value) {
+        return value === 'isFtuRunning' ? true : false;
```
Comment (fcampo) [inline apps/system/test/unit/notifications_test.js]: shouldn't `return value === 'isFtuRunning';` be enough?

Comment (etiennesegonzac) [inline apps/system/test/unit/notifications_test.js]: nit: please change to the more readable
```
    var query = this.sinon.stub(Service, 'query')
    query.withArgs('isFtuRunning').returns(true);
```

**Comment by try-server-hook** [comment]
[mancas Manuel Casas (mancas)](https://github.com/mancas) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=84eb79cbadc9)

**Comment by try-server-hook** [comment]
[mancas Manuel Casas (mancas)](https://github.com/mancas) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=743d9be02c7d)

**Comment by try-server-hook** [comment]
[mancas Manuel Casas (mancas)](https://github.com/mancas) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d3afd57960d3)

**Comment by try-server-hook** [comment]
[mancas Manuel Casas (mancas)](https://github.com/mancas) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=81a369f3e6df)

**Comment by try-server-hook** [comment]
[mancas Manuel Casas (mancas)](https://github.com/mancas) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=942812acab60)

**Comment by try-server-hook** [comment]
[mancas Manuel Casas (mancas)](https://github.com/mancas) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2b5ea4e6836d)

**Comment by try-server-hook** [comment]
[mancas Manuel Casas (mancas)](https://github.com/mancas) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=30c34d154fdf)

**Pull request merged**